server/lib/amplitude.js

```diff
@@ -162,15 +156,18 @@ function receiveEvent (event, request, data) {
 
   const eventType = event.type;
   let mapping = EVENTS[eventType];
-  let eventCategory;
+  let eventCategory, eventTarget;
```

Comment:
I imagine someone may come along and say that `eventTarget` is confusing nomenclature. But these are clearly nothing to do with DOM events and I couldn't think of another suitable-and-generic name for the way I'm using this property. Happy to rename it if there's suggestions though.